komodo/release_cleanup.py

```diff
@@ -180,11 +180,13 @@ def run_cleanup(args, parser):
 def run_prettier(args, _):
     release_files = [filename for sublist in args.files for filename in sublist]
 
-    if not args.check or all(
-        [prettified_yaml(filename, args.check) for filename in release_files]
+    if all(
+        [prettified_yaml(filename, check_only=args.check) for filename in release_files]
     ):
         sys.exit(0)
-
+    if args.check is False:
+        # The prettifier has successfully reformatted all files
```

Comment:
Don't we need a step here to prettify?

Reply:
Added a comment to make it a little bit clearer. I have thought about refactoring to increase readability but did not come up with a good (/easy) solution.

Reply:
Change the name of the argument from `check` to `check-only`. I think that improves readability.

Reply:
Do you know if this imply any need of `deprecation warning`?

Reply:
I chose not to, since it would be an API break that might not be very important to do. I am not sure if we within argparse are able to deprecate option variants.